kzg/src/msm/precompute.rs

```diff
@@ -8,12 +8,16 @@ use crate::{Fr, G1Affine, G1Fp, G1GetFp, G1Mul, G1ProjAddAffine, G1};
     all(feature = "arkmsm", feature = "bgmw"),
     all(feature = "arkmsm", feature = "sppark"),
     all(feature = "arkmsm", feature = "wbits"),
+    all(feature = "arkmsm", feature = "bos_coster"),
     all(feature = "bgmw", feature = "sppark"),
     all(feature = "bgmw", feature = "wbits"),
-    all(feature = "sppark", feature = "wbits")
+    all(feature = "bgmw", feature = "bos_coster"),
+    all(feature = "sppark", feature = "wbits"),
+    all(feature = "sppark", feature = "bos_coster"),
+    all(feature = "wbits", feature = "bos_coster"),
```

Comment:
no need to check this, bos_coster is compatible with all other features (except arkmsm, but you can ignore this for now)